Stabilize named callback arguments (#6854)
slint-ui/slint: docs/src/content/docs/concepts/callbacks.md

}
```

Callback arguments can also have names.

> The names of argument have currently no semantic value, but they can be usefull in tooling or auto-completion.

Reviewer: ```suggestion
The names of arguments have currently no semantic value, but they improve readability of your code.
```

```slint
export component Example inherits Rectangle {

>     // Declares a callback with named argument

Reviewer: ```suggestion
    // Declare a callback with named argument
```

    callback hello(foo: int, bar: string);

>     // But the names can still be overriden with
>     // anything when setting a handler

Reviewer: ```suggestion
    // The names can be overridden with
    // anything when setting a handler
```
